hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java

```diff
@@ -29,7 +30,7 @@
  */
 public class JsonPathCursor {
 
-    private static final Charset UTF8_CHARSET = Charset.forName("UTF8");
+    private static final Charset UTF8_CHARSET = StandardCharsets.UTF_8;
```

Comment:
I believe we don't need the static field at all if we can reuse the `StandardCharsets.UTF_8`

Reply:
I agree with you. If you want I can remove the static field.

Reply:
Yeah, that's the right way to go. There's no use in having a static field referencing another constant

Reply:
I removed redundant UTF8_CHARSET static field. 
https://github.com/hazelcast/hazelcast/pull/17741/commits/627d61ae28bb86249fee966c17878dc7a89e1895